types/office-runtime/index.d.ts

```diff
@@ -23,85 +26,132 @@ declare namespace OfficeRuntime {
   const storage: Storage;
   /**
    * Asynchronous, global, and persistent key-value storage.
+   *
+   * [Api set: ExcelApi 1.9]
```

Comment:
Storage APIs are custom functions only. Excel add-ins can't access them...

Reply:
thanks! Fixed in latest commit.